src/core/server/base.js

```diff
@@ -198,10 +199,11 @@ function baseServer(
     });
   }
 
-  // Following the ops monitoring convention, return version info at this URL.
+  // Following the ops monitoring Dockerflow convention, return version info at
+  // this URL. See: https://github.com/mozilla-services/Dockerflow
   app.get('/__version__', viewVersion);
```

Comment:
Why doesn't this use `viewFrontendVersionHandler()`? I don't think we should maintain separate implementations.

Reply:
Alrighty, I can add it here too.